File: WebDriverTests_Csharp/Autotest.Core/Autotests.Utilities/WebTestCore/SystemControls/HtmlControlContainer.cs
using OpenQA.Selenium;

namespace Autotests.Utilities.WebTestCore.SystemControls
{
    public class HtmlControlContainer : HtmlControl
    {
        public HtmlControlContainer(By locator, HtmlControl container = null)
            : base(locator, container)
        {
        }

        public HtmlControlContainer(string idLocator, HtmlControl container = null)
            : base(idLocator, container)
        {
        }
    }
}

File: WebDriverTests_Java/WebDriverTests/src/test/java/core/systemControls/HtmlControlContainer.java
package core.systemControls;


import org.openqa.selenium.By;

public class HtmlControlContainer extends HtmlControl
{

    public HtmlControlContainer(By locator)
    {
        this(locator, null);
    }

    public HtmlControlContainer(By locator, HtmlControl container)
    {
        super(locator, container);
    }


    public HtmlControlContainer(String idLocator)
    {
        this(idLocator, null);
    }

    public HtmlControlContainer(String idLocator, HtmlControl container)
    {
        super(idLocator, container);
    }
}
